src/index.html

```diff
@@ -17,16 +17,40 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <body>
+      <div class="container">
+        <form
+          class="search-bar input__big"
+          data-qa="big"
+        >
+          <input
+            class="input input__big search-bar__input"
```

Comment:
```suggestion
          class="search-bar input__big"
          data-qa="big"
        >
          <input
            class="input input__big"
```
redundant - add specific for big input styles to `input__big` class. Shared styles add to `input` class